web-console/4.0-designs/forms/forms.md

```diff
@@ -0,0 +1,38 @@
+# Forms
+
+## Labels
+
+* Labels should be written in headline style.
+* Labels should be top-aligned.
+* Required fields should be identified with a red asterisk to the right of the label.
+
+![Labels example](img/service.png)
+
+## Buttons
+
+* Button labels should be written in headline style.
+* In embedded forms, confirmation buttons should be left aligned at the bottom of the page. Cancel buttons should be to the right of confirmation buttons.
```

Comment:
Can we specify for side panel forms too? Even if we don't have them now we may in the future. Something like: 

In embedded forms AND side panel forms, confirmation buttons should be left aligned at the bottom of the page. Cancel buttons should be to the right of confirmation buttons.